portfolio/src/main/webapp/script.js

```diff
@@ -68,7 +70,44 @@ const appendComments = (comments, commentContainerElement) => {
     const userElement = document.createElement("h3");
     userElement.innerHTML = comments[commentId].user;
 
+    const votesContainerElement = document.createElement("div");
```

Comment:
According to the style guide, string literals are delimited with single quotes ( ' ).